rx.lua

```diff
@@ -260,6 +260,26 @@ function Observable:dump(name, formatter)
   return self:subscribe(onNext, onError, onCompleted)
 end
 
+--- Creates an Observable that produces values from a file, line by line.
+-- @arg {string} filename - The name of the file used to create the Observable
+-- @returns {Observable}
+function Observable.fromFileByLine(filename)
+  return Observable.create(function(observer)
+    local f = io.open(filename, "rb")
```

Comment:
indeed, this is the purpose of this line, with the `if f` next (I forgot it)
but anyway, the `binary` parameter should not be needed